benefits/core/templates/core/includes/modal-trigger.html

```diff
@@ -1,2 +1,4 @@
-<!-- Button to trigger modal -->
-<button type="button" class="{{ classes }}" data-bs-toggle="modal" data-bs-target="#{{ id }}">{{ text }}</button>
+<!-- Inline link to trigger modal -->
+{# djlint:off #}
+<a href="#{{ id }}" class="{{ classes }}" id="{{ id }}" data-bs-toggle="modal" data-bs-target="#{{ modal }}">{{ text }}{% if login %}<span class="fallback-text color-logo">Login.gov</span>{% endif %}</a>{% if period %}.{% endif %}
```

Comment:
I think this needs to be `href="#{{ modal }}"` based on the renamed include variable?

Since `{{ id }}` is the ID of this link.

Reply:
https://github.com/cal-itp/benefits/pull/1627/commits/8392b1289ae32daa1f6f71c6d5054f6c3779fdd9